plugins/toolkit/jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/listeners/CodeWhispererCodeScanFileListener.kt

```diff
@@ -15,7 +15,7 @@ import software.aws.toolkits.jetbrains.services.codewhisperer.util.CodeWhisperer
 internal class CodeWhispererCodeScanFileListener(val project: Project) : EditorFactoryListener {
     override fun editorCreated(event: EditorFactoryEvent) {
         val actionManager = CodeWhispererExplorerActionManager.getInstance()
-        if (event.editor.virtualFile.isFile && event.editor.project == project && actionManager.isAutoEnabledForCodeScan() &&
+        if ( event.editor.project == project && actionManager.isAutoEnabledForCodeScan() &&
```

Comment:
don't we still need to check if it's a file?

Reply:
The check is added in later for file and validity of file.